hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/KafkaMessageReceiverFactory.java

```diff
@@ -168,14 +168,10 @@ private KafkaConsumer<byte[], byte[]> createKafkaConsumer(Topic topic, Subscript
     }
 
     private void addKafkaAuthorizationParameters(Properties props) {
-        if (kafkaAuthorizationParameters.isEnabled()) {
-            props.put(SASL_MECHANISM, kafkaAuthorizationParameters.getMechanism());
-            props.put(SECURITY_PROTOCOL_CONFIG, kafkaAuthorizationParameters.getProtocol());
-            props.put(SASL_JAAS_CONFIG,
-                    "org.apache.kafka.common.security.plain.PlainLoginModule required\n"
-                            + "username=\"" + kafkaAuthorizationParameters.getUsername() + "\"\n"
-                            + "password=\"" + kafkaAuthorizationParameters.getPassword() + "\";"
-            );
+        if (kafkaParameters.isEnabled()) {
```

Comment:
It should be named like `isAuthenticationEnabled()`.  It is just the interface method so can be changed freely without worry about backward compatibility